opl/generators/README.md

```diff
@@ -106,3 +106,13 @@ Example usage:
 Generate Chrome notifications Kafka messages.
 
 Just supply the count of messages to be generated.
+
+`create_packages_template_from_dnf_repoquery.py`
+---------------------------------------
+
+This is just a standalone helper script to help you process `dnf repoquery` into a generator-friendly structure for packages.
+
+Before using this, run `dnf repoquery --available --show-duplicates` on your desired machine/VM to get a list of all packages that could be installed on your mahcine.
```

Comment:
`dnf repoquery --available --queryformat="%{name} %{epoch} %{version} %{release} %{arch} \n"` might be easir to process (`dnf` on my Fedora does not support `--show-duplicates` so have not tried that, but it is still important).